src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonReaderTests.cs

```diff
@@ -4534,5 +4535,14 @@ public static IEnumerable<object[]> GetCommentUnescapeData
                 return dataList;
             }
         }
+
+        public static IEnumerable<object[]> InvalidDateTimePropertyValueData()
```

Comment:
Nit: shouldn't this helper method be collocated with the source file of the test?

Reply:
Removed this method as I have used "InlineData" attribute now (more in the discussion below).